Directory.Build.props

```diff
@@ -7,6 +7,13 @@
     <EnableSourceControlManagerQueries>false</EnableSourceControlManagerQueries>
   </PropertyGroup>
   
+  <ItemGroup>

```

Comment:
Why EnableSourceLink is disabled? we disabled only for non windows os but will be enabled by default

Reply:
> Why EnableSourceLink is disabled? we disabled only for non windows os but will be enabled by default

Sorry. I do not understand your question and statement. Did I unintentionally changed the state of `EnableSourceLink`?

Reply:
I'm sorry sure I need to improve my english :-)

Well, the first lines on this file remove the source link, but after check the history the problem is not your. Can yoy remove the initial  property group?

  <!-- Temporal - Waiting for a netcoreapp 3.0 version of Github Source Link -->
  <PropertyGroup>
    <EnableSourceLink>false</EnableSourceLink>
    <EnableSourceControlManagerQueries>false</EnableSourceControlManagerQueries>
  </PropertyGroup>

Reply:
@unaizorrilla is this change OK for you?

Reply:
looks good, I try to merge tomorrow!